apps/fxc-front/src/app/logic/score/league/czech.ts

```diff
@@ -27,6 +28,7 @@ abstract class CZXCBase {
 
 export class CzechLocal extends CZXCBase {
   name = 'Czech (ČPP local)';
+  code: LeagueCode = 'czl';
```

Comment:
There might be a better way to do this - see how it's done for the tracker at https://github.com/vicb/flyxc/blob/2b869f5bc06ce845e5b58f84642ab3e7cd2f044b/libs/common/src/lib/live-track.ts#L37 and up

Reply:
Not sure it is the same thing... or may be I dont understand. I added some comments in code. Tell me if it's ok or if I am missing something.